include/bluetooth/mesh/time_cli.h

```diff
@@ -57,6 +129,13 @@ struct bt_mesh_time_cli {
 	struct bt_mesh_model_pub pub;
 	/** Acknowledged message tracking. */
 	struct bt_mesh_model_ack_ctx ack_ctx;
+	/** Collection of handler callbacks.
+	*
+	* @note Function pointers to handlers must point to valid memories,
+	* which have execute permissions.
+	*
+	*/
```

Comment:
```suggestion
	/** Collection of handler callbacks.
	 *
	 * @note Function pointers to handlers must point to valid, executable 
	 * memory.
	 */
```

Reply:
I think the documentation here is somewhat misleading. 
The thing we wish to document here is that the structure is not copied right and that storage of this struct is up to the application, right?
This storage does not have to be in executable memory.

I think "Must point to memory that remains valid." is good enough.

Placing the function that the function pointer points to is really up to the compiler, so not something that the user of the API should have to care about.

Reply:
Last time I misunderstood the idea. Yes, indeed, I agree with you. And I have fixed it.